### PR TITLE
Feature capture tag removal

### LINKED ISSUE #675: Capture Details: Allow user to remove capture tags
If a user has the `APPROVE_TREE` policy, they should be able to remove capture tags, one by one, from captures via the Capture Details dialog:

1. Capture tag chips (the last ones listed under _Other_) are marked deletable for privileged users (X appears to the right of the label)
2. Clicking the X prompts _Are you sure you wish to remove {name} from this capture?_
3. User clicks _Remove Tag_ to confirm (_Cancel_ option also available).
4. Tag is removed from the capture by calling `DELETE /api/tree_tags/:id`.
5. Tag disappears from the list of chips.

Screenshot 2022-06-02 at 14 20 55

### PR DESCRIPTION
**Pull request by ghost**
## Description

Fixed the tag removal in the capture detail panel to work with new api.

**Issue(s) addressed**

- Resolves #675 

**What kind of change(s) does this PR introduce?**

- [ ] Enhancement
- [x] Bug fix
- [ ] Refactor

**Please check if the PR fulfils these requirements**

- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Issue

**What is the current behavior?**
- Capture still appears when searching captures with removed tag.
- Capture still contains the removed tag in its field "tags"

**What is the new behavior?**
- After removing a tag, the capture do not appear in the search results that filters by the removed tag
- After removing a tag, capture do not contain the removed tag in its tags field

## Breaking change

**Does this PR introduce a breaking change?**

### CODE REVIEW

**Comment by maxt0214** [comment]
@nmcharlton For the issue of capture still appears after tag is removed. I believe this is a backend issue. When I restart the frontend over and filter captures by tag 'test 1', the capture that had 'test 1' removed yesterday still appears.

**Review by nmcharlton — CHANGES_REQUESTED** [review]
Can you take a look at the failing test and see if it's something you're able to fix? `npm run test`

File: src/context/CaptureDetailContext.js [hunk 1]
```diff
@@ -59,6 +63,17 @@ export function CaptureDetailProvider(props) {
     }
   };
 
+  const getCaptureTags = async (captureId) => {
+    try {
+      api.getCaptureTags([captureId]).then((tags) => {
+        console.log('All tags: ', tags);
```
[the remark this answers is not in the document]
Comment (maxt0214) [reply src/context/CaptureDetailContext.js:69]: Will do

[the remark this answers is not in the document]
Comment (maxt0214) [reply src/context/CaptureDetailContext.js:73]: I have removed this log. Its only a temporary debugging log

**Comment by maxt0214** [comment]
@nmcharlton I have fixed the issue and reran the tests. Everything works on local.

**Review by nmcharlton — APPROVED** [review]
Thanks @maxt0214, great work!

**Pull request merged**